src/directives/rccif.js

```diff
@@ -0,0 +1,97 @@
+directives.directive('rcCif', [function(){
+	return {
+		require:"ngModel",
+	
+	 link: function (scope, elm, attrs, ctrl){
+		var validationToExecute = function (viewValue) {
+			var currentTypedValue = viewValue;
+
```

Comment:
No need to create a variable for this; just feed viewValue directly to fullCIFValidation